eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -67,7 +67,7 @@ stages:
                         Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
                       displayName: Stage artifacts
                       timeoutInMinutes: 5
-                    - template: eng/pipelines/templates/steps/create-tags-and-git-release.yml@azure-sdk-tools
```

Comment:
Can we now remove the reference to azure-sdk-tools from the ci.yml files?

Reply:
we still use it [here](https://github.com/Azure/azure-sdk-for-net/blob/586735bd747ee249290aa25589728a536c14baad/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L12) and some other places in `archetype-sdk-client.yml` but those can probably also be moved into eng/common

Reply:
Cool. Yes I'd like those to be updated to use eng/common, please file a issue for those if we don't already have one.